org.opent2t.sample.multisensor.superpopular/com.wink.sensorpod/js/thingTranslator.js

```diff
@@ -70,10 +70,15 @@ class StateReader {
 }
 
 function convertDeviceDateToTranslatorDate(unixTimestamp) {
-    // Date takes a number of milliseconds, so convert seconds to milliseconds
-    var datetime = new Date(unixTimestamp * 1000);
+    // Only convert if input is valid numeric value
+    if (unixTimestamp && !isNaN(unixTimestamp)) { 
+        // Date takes a number of milliseconds, so convert seconds to milliseconds
+        var datetime = new Date(unixTimestamp * 1000);
+        return datetime.toISOString();
+    }
 
-    return datetime.toISOString();
+    // return all other invalid input as-is since we can't convert to DateTime
+    return unixTimestamp;
```

Comment:
Well, the standard is in our schema, not theirs. We should be responsible for ensuring we conform to our schema.

I'm suggesting that we initially try to use the _changed_at property (e.g. motion_changed_at) since that is the time that is reported by the physical device itself, and if Wink doesn't provide that for whatever reason (which Arjun saw in his test), we can fall back to the _updated_at property, which is set by Wink, rather than reported by the device.

If there are cases where neither of those properties exist, I would suggest we fall back to a valid ISO8601 datetime (e.g. the epoch time, or something similar), rather than returning something that doesn't conform to our schema.